pipeline-model-definition/src/test/resources/parallelStagesFailFast.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+pipeline {
+    agent none
+    stages {
+        stage("foo") {
+            failFast true
```

Comment:
Just to be clear: `failFast false` is the default, is there a reason to ever set that? could this just be `failFast` ?

Reply:
(I was kinda hoping, and maybe it's stupid, to have a syntax like:
```
parallel failFast {
...
```
)

Reply:
Yeah, that's stupid and not happening for any number of reasons. =)

First and foremost, we can't do `failfast` as a "field" on its own because then it's not a method call and things blow up in parsing. And to do something like `parallel failFast { ... }`, we'd need to have a "variable" `failFast` defined to make that work syntactically, a la `agent none` and `agent any`, and I ain't goin' down that road again. =)